ignition/what-is-ignition.md

```diff
@@ -0,0 +1,56 @@
+# What is Ignition? #
+
+Ignition is a new provisioning utility designed specifically for CoreOS. At the
+the most basic level, it is a tool for manipulating disks during early boot.
+This includes partitioning disks, formatting partitions, writing files (regular
+files, systemd units, networkd units, etc.), and configuring users. On first
+boot, Ignition reads its configuration from a source-of-truth (remote URL,
+network metadata service, hypervisor bridge, etc.) and applies the configuration.
+
+## Ignition vs coreos-cloudinit ##
```

Comment:
The legal/technical term. I don't want users confusing this for Canonical's CloudInit.